LoopKitUI/View Controllers/DismissableHostingController.swift

```diff
@@ -8,7 +8,6 @@
 
 import SwiftUI
 
-
 public class DismissibleHostingController: UIHostingController<AnyView> {
```

Comment:
I think we should rename this to just 'HostingController' to make it clear that it serves as our main bridge to SwiftUI for all of our custom environment data (which was originally just `dismiss`)